cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java

```diff
@@ -101,12 +109,23 @@ public String getFormat() {
 
     @Override
     public Network importData(ReadOnlyDataSource ds, NetworkFactory networkFactory, Properties p) {
+        if (profiling != null) {
+            profiling.start();
+        }
```

Comment:
You should use the same design: a lazy implementation or check in CgmesExport that the profiling is not null. Try to be consistent in both classes

Reply:
As Luma commented, we will remove profiling from the code.

Reply:
Profiling code is cleaned out